src/components/textarea/tests/TextArea.spec.tsx

```diff
@@ -77,6 +77,15 @@ describe('TextArea', () => {
             expect(onChange).toHaveBeenCalledTimes(1);
         });
 
+        it('should call prop onChangeCallBack on textarea change', () => {
+            const onChangeCallBack = jasmine.createSpy('onChangeCallBack');
+
+            wrapper.setProps({onChangeCallBack});
+            textArea.simulate('change');
+
+            expect(onChangeCallBack).toHaveBeenCalledTimes(1);
+        });
+

```

Comment:
You'll need to add tests calling `handleOnChange` without `onChange` and `onChangeCallBack` and check that it does not throw or else the coverage will go down.

Reply:
I tried to do something for the onChange not defined but the not.throw want a function as argument. I check for an other solution to test.